doc/tutorials/java-kdb.md

```diff
@@ -89,3 +89,8 @@ try (KDB kdb = KDB.open(key)) {
 First we create a new `KDB` object and fetch all keys for the desired namespace, in this example the `user` namespace. Since it saves all
 keys in our passed `set` variable we can then iterate through it by a simple for loop.
 The `at(int)` method gives us the key on the corresponding position which we will print out in this example.
+
+### Read Multiple Keys From KDB
+
+[This](../../examples/external/java/read-keys-example) example shows how to read multiple keys. It provides comments for further clarification.
```

Comment:
Please describe how to build and execute the jar. Also shortly describe what it does (for people that want to execute it).